builtin/client/register.lua

```diff
@@ -4,6 +4,11 @@ core.callback_origins = {}
 local getinfo = debug.getinfo
 debug.getinfo = nil
 
+-- Runs given callbacks
```

Comment:
An additional `-` is needed to make this an LDoc/LuaDoc comment

Reply:
Done.
Those `@param` tags were actually just pseudostuff. Now they are used properly.

Btw. builtin/common/misc_helpers has lines with 80 `-`s. But it does not really look as if that's there for ldoc.